data/CA/incentives.json

```diff
@@ -0,0 +1,2817 @@
+    "short_description": {
+      "en": "$100-$750 rebate for qualifying Induction Cooktop/range."
+    }
+  },
+  {
+    "id": "CA-112",
+    "authority_type": "utility",
+    "authority": "ca-sacramento-municipal-utility-district",
+    "payment_methods": [
+      "rebate"
+    ],
+    "item": "heat_pump_water_heater",
+    "program": "ca_sacramentoMunicipalUtilityDistrict_applianceRebates",
+    "amount": {
+      "type": "dollar_amount",
+      "number": 3000,
+      "minimum": 500,
+      "maximum": 3000
+    },
+    "owner_status": [
+      "homeowner"
+    ],
+    "short_description": {
+      "en": "Rebates up to $3,000 for qualifying Heat Pump Water Heater."
+    },
+    "bonus_available": true
+  },
+  {
+    "id": "CA-113",
+    "authority_type": "utility",
+    "authority": "ca-silicon-valley-power",
+    "payment_methods": [
+      "rebate"
+    ],
+    "item": "ebike",
+    "program": "ca_siliconValleyPower_rebates",
+    "amount": {
+      "type": "dollar_amount",
+      "number": 0.1,
+      "maximum": 500
+    },
+    "owner_status": [
+      "homeowner",
+      "renter"
+    ],
+    "short_description": {
+      "en": "Up 10% rebate for the purchase of an electric bicycle (e-bike), up to $300."
+    },
+    "bonus_available": true,
+    "low_income": "ca-silicon-valley-power"
+  },
+  {
+    "id": "CA-114",
+    "authority_type": "utility",
+    "authority": "ca-silicon-valley-power",
+    "payment_methods": [
+      "rebate"
+    ],
+    "item": "heat_pump_water_heater",
+    "program": "ca_siliconValleyPower_rebates",
+    "amount": {
+      "type": "dollar_amount",
+      "number": 500,
+      "minimum": 500,
+      "maximum": 1000
+    },
+    "owner_status": [
+      "homeowner"
+    ],
+    "short_description": {
+      "en": "$500 rebate for the installation of an electric heat pump water heater if replacing an electric resistance water heater."
+    }
+  },
+  {
+    "id": "CA-115",
+    "authority_type": "utility",
+    "authority": "ca-silicon-valley-power",
+    "payment_methods": [
+      "rebate"
+    ],
+    "item": "electric_vehicle_charger",
+    "program": "ca_siliconValleyPower_rebates",
+    "amount": {
+      "type": "dollar_amount",
+      "number": 550
+    },
+    "owner_status": [
+      "homeowner"
+    ],
+    "short_description": {
+      "en": "Up to $550 rebate for the installation of an electric vehicle charging station."
+    }
+  },
+  {
+    "id": "CA-116",
+    "authority_type": "utility",
+    "authority": "ca-silicon-valley-power",
+    "payment_methods": [
+      "rebate"
+    ],
+    "item": "rooftop_solar_installation",
+    "program": "ca_siliconValleyPower_rebates",
+    "amount": {
+      "type": "dollars_per_unit",
+      "number": 3.75,
+      "unit": "kilowatt",
+      "maximum": 11250
+    },
+    "owner_status": [
+      "homeowner"
+    ],
+    "short_description": {
+      "en": "Income-Qualified Solar Grant Program provides a grant of $3.75 per watt, up to 3 kilowatts DC."
```

Comment:
Unit confusion here. The `unit` should be `watt` if that's how the utility is presenting it. If they're presenting it as kW, then the unit should be `kilowatt` and the number should be 3750.